### ISSUE TITLE
Remap OSU repo and associated namespace(s)

### ISSUE THREAD

**Issue by LocoDelAssembly** [sfg-taxonpages/orthoptera#26]
OSU was mapped into Ohio State University, however this is the case for Plecoptera SF only, for Orthoptera SF it should have been Oklahoma State University.

No other SFs seem to use this coden. Existing Oklahoma repo probably requires update to use OSUS instead ( https://www.gbif.org/grscicoll/institution/d412f3cf-7ea7-4859-a8a7-0947b87b3d64 ) @mjy? It might happen that in the past both used same coden, in fact today there are still two using OSU:
- Ohio State University: https://www.gbif.org/grscicoll/institution/bed5c349-5b3a-4c52-bc4a-16d7297cb1af
- Oregon State University, Department of Fisheries Wildlife's Birds, Mammals, and Fishes Collections: https://www.gbif.org/grscicoll/institution/64fb4b4c-3966-4b9f-94e0-02be5b542470

```sql
SELECT *
FROM tblDepos
WHERE Coden = 'OSU';
```
```
DepoID	PrefID	FileID	Coden	ShortName	FullName	Link	LastUpdate	ModifiedBy	CreatedOn	CreatedBy
912	912	1	OSU	Stillwater	K. C. Emerson Museum, Oklahoma State University		2013-02-12 18:32:00	-1	2013-02-12 18:32:00	-1
1612	1612	11	OSU	Ohio State University	USA, Ohio, Columbus, Ohio State University		2013-03-12 16:33:00	-1	2013-03-12 16:33:00	-1
```

**Comment by mjy** [comment]
Create 2 namespaces if they use duplicate and we want to maintain historical reference to the identifier (not REpository!) Differentiate the two for picking one or the other in the name, and both have "OSU" as the verbatim.

**Comment by LocoDelAssembly** [comment]
Apparently only two specimens in OSF, but for some reason them are pointing to OSEC instead? Maybe you edited @mabecabrera ? (Although last update is still 2011 by Sam Heads).

https://sfg.taxonworks.org/collection_objects/1544915
https://sfg.taxonworks.org/collection_objects/1544916

When displaying related objects for `OSU (SF IMPORT DUPLICATE)` there is nothing shown other than Marilyn user creating the namespace.

**Comment by mabecabrera** [comment]
No. II haven't assigned myself any different acronyms. I have not used the verbatim either. Should I? 
What do we do in this case that both collections use the same acronym.

**Comment by mabecabrera** [comment]
I think this is another example for the same thing.

This is from the Excel list.
National Museum of Natural History, Smithsonian Institution (USNM)(SF IMPORT DUPLICATE)
but:
<img width="742" alt="image" src="https://github.com/sfg-taxonpages/orthoptera/assets/63611746/2393dd40-a5e1-49fc-bb8f-287b47886b4c">
